firefox-ios/Client/Frontend/Settings/Main/AppSettingsTableViewController.swift

```diff
@@ -358,6 +359,7 @@ class AppSettingsTableViewController: SettingsTableViewController,
             FasterInactiveTabs(settings: self, settingsDelegate: self),
             OpenFiftyTabsDebugOption(settings: self, settingsDelegate: self),
             FirefoxSuggestSettings(settings: self, settingsDelegate: self),
+            FeatureFlagsSettings(settings: self, settingsDelegate: self)
```

Comment:
@cyndichin Do we want to expose this in app store builds? Or is this primarily for dev/testing purposes? (In which case we might want to wrap it in `#if DEBUG` or something similar)

LMK if I'm forgetting/misunderstanding, but AFAIK this menu is available even for app store users and I wasn't sure if we wanted to allow everyone to easily switch experiments on and off at will.

Reply:
lol I asked this question too.

Reply:
Yes, you're correct! We don't want to so I can add `MOZ_CHANNEL_BETA` as well. This is primary for testing purposes to help users test in nightly.

Reply:
@mattreaganmozilla @adudenamedruby curious on the context, do you know why we expose the debug menu in app store build?

Reply:
@cyndichin At minimum I know it's been useful to allow users to copy out their log files, and there's a few other debug tools in there we've asked users to use in the past to help us get diagnostics.

Reply:
great, thank you for clarifying!